cypress/e2e/signIn.cy.js

```diff
@@ -1,7 +1,16 @@
 /// <reference types="cypress" />
 
 describe('Sign In page', () => {
+  beforeEach(() => cy.visit('https://conduit.mate.academy/user/login'));
+
   it('should provide an ability to log in', () => {
+    cy.url().should('contain', 'user/login');
+
+    cy.get(':nth-child(1) > .form-control').type('fortest123123@gmail.com');
+    cy.get(':nth-child(2) > .form-control').type('Qwerty123.');
```

Comment:
It's recommended to use `data-cy` or similar attributes for selecting elements to make the tests more resilient to changes in the markup. Using CSS selectors like `:nth-child` can lead to brittle tests that break when the layout changes.